source/extensions/request_id/uuid/config.cc

```diff
@@ -16,7 +16,7 @@ void UUIDRequestIDExtension::set(Http::RequestHeaderMap& request_headers, bool e
   const Http::HeaderEntry* request_id_header = request_headers.RequestId();
 
   // No request ID then set new one anyway.
-  if (request_id_header == nullptr) {
+  if (request_id_header == nullptr || request_id_header->value().empty()) {
```

Comment:
Here is a minor behavior change. I think this is fine because empty request id make no sense and it's fine to overwrite it directly.

But still ping @envoyproxy/runtime-guard-changes to see if runtime guard is necessary or not.

Reply:
Do I need to ping @envoyproxy/runtime-guard-changes by myself somewhere?

Reply:
Two days gone and no one object, then I will make decision by myself.